### ISSUE TITLE
My acc is marked as a bot.

### ISSUE THREAD

**Issue by InariShino** [CommanderRoot/twitch-tools#69]
Hi! I noticed my acc was marked as a known bot. I've been blocked by some streamers cause of it. Is there any way to revert it?

**Comment by CommanderRoot** [comment]
Duplicate of #30 

Your account was added to the list on 2022-09-25. This most likely happened because it was part of one or multiple follow bot raids. It's very likely that it was compromised back then.
Either way it happened I have now removed it from the list

**Comment by InariShino** [comment]
Thank you so much for the help!!! 🥹❤️
